specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionsListConnectionKeys.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
let's not document the listConnectionKeys API. There was a scenario we added it specifically for, but it has never seen the light of day, so we should probably deprecate this API (as we don't want users to actually use it; with connection key they can use the connection bypassing our runtime, and thus without being billed)

Reply:
Removed the example and the endpoint from the swagger